ethereum-financial-v2/src/services/aave/Aave.srv.ts

```diff
@@ -0,0 +1,173 @@
+import { Finding, FindingSeverity, FindingType } from 'forta-agent'
+import { BlockDto } from '../../entity/events'
+import { Logger } from 'winston'
+import BigNumber from 'bignumber.js'
+import * as E from 'fp-ts/Either'
+import { networkAlert } from '../../utils/errors'
+import { elapsedTime } from '../../utils/time'
+
+export interface IAaveClient {
+  getStethBalance(address: string, blockNumber: number): Promise<E.Either<Error, BigNumber>>
+
+  getTotalSupply(blockNumber: number): Promise<E.Either<Error, BigNumber>>
+
+  getStableDebtStEthTotalSupply(blockNumber: number): Promise<E.Either<Error, BigNumber>>
+
+  getVariableDebtStEthTotalSupply(blockNumber: number): Promise<E.Either<Error, BigNumber>>
+}
+
+const GWEI_DECIMALS = new BigNumber(10).pow(9)
+const ASTETH_ETH_1 = GWEI_DECIMALS.times(1)
+// 12 hours
+const HOURS_12 = 60 * 60 * 12
```

Comment:
```suggestion
const MINUTES_10 = 60 * 10
```

should repeat critical alerts left here more frequently